src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/SettingsEvoSuiteComponent.kt

```diff
@@ -38,27 +39,26 @@ class TestGenieSettingsComponent {
 
     init {
         panel = FormBuilder.createFormBuilder()
```

Comment:
Maybe it is worth refactoring the contents of the init function. You can take a look at the way `init{}` is done in the TestGenieToolWindow. 

I think this would make the code more clear.

Reply:
Will assess

Reply:
Thank you.